client/dashboard/sites/settings-database/reset-password-modal.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Button,
+	Modal,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import { resetPhpMyAdminPassword } from '../../data';
+
+interface ResetPasswordModalProps {
+	siteSlug: string;
+	onClose: () => void;
+	onSuccess: () => void;
+	onError: () => void;
+}
+
+export default function ResetPasswordModal( {
+	siteSlug,
+	onClose,
+	onSuccess,
+	onError,
+}: ResetPasswordModalProps ) {
+	const [ isRestoring, setIsRestoring ] = useState( false );
+
+	const handleRestore = async () => {
```

Comment:
Nit: there is no `await` here for the `async`

Reply:
Good catch, removed the leftover code in c404fa475b71b5750ff34b40d7a1fcdb7c787c35.